sdk/formrecognizer/Azure.AI.FormRecognizer/src/ExtractedTableCell.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Azure.AI.FormRecognizer.Models
+{
+    public class ExtractedTableCell
+    {
+        internal ExtractedTableCell(DataTableCell_internal dataTableCell, ReadResult_internal readResult, ICollection<string> references)
+        {
+            BoundingBox = new BoundingBox(dataTableCell.BoundingBox);
+            ColumnIndex = dataTableCell.ColumnIndex;
+            ColumnSpan = dataTableCell.ColumnSpan.HasValue ? dataTableCell.ColumnSpan.Value : 1;
+            Confidence = dataTableCell.Confidence;
+            IsFooter = dataTableCell.IsFooter.HasValue ? dataTableCell.IsFooter.Value : false;
```

Comment:
Why are all the values on original type nullable?

Reply:
Because they might not be returned from the service.

@ctstone is it correct to default these values?  I believe I did this because you had done it in a table method in the original .NET library.  Can you confirm the semantics of these coming back null?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/10448